ratis-test/src/test/java/org/apache/ratis/util/TestTraditionalBinaryPrefix.java

```diff
@@ -17,15 +17,18 @@
  */
 package org.apache.ratis.util;
 
-import org.junit.Test;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.Timeout;
 
 import static org.apache.ratis.util.TraditionalBinaryPrefix.long2String;
 import static org.apache.ratis.util.TraditionalBinaryPrefix.string2long;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.fail;
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
This test is failing since the `assertEquals` parameters have a different order in unit 5.